src/app/components/layout/certificate-base-login/certificate-base-login.component.html

```diff
@@ -6,28 +6,20 @@
       <button class="delete" aria-label="close" (click)="toggleModal(false)"></button>
     </header>
     <section class="modal-card-body">
```

Comment:
Add some padding top and bottom in case of showBrowseCertificate, so that we don't see a jumping effect?